### ISSUE TITLE
Prevent save/publish if ContrastChecker is showing warning/error

### ISSUE THREAD

**Issue by firstdivision** [WordPress/gutenberg#34701]
## What problem does this address?
When editing a post, if the user selects colors that activate the `ContrastChecker`'s WGAG warning, I would like an option to have this be treated as an error that prevents the post from being saved/published. Currently it only displays a warning/notice in the inspector panel.

## What is your proposed solution?
Add an option somewhere in WP admin to edit the behavior of the ContrastCheckers and whether it is treated as a warning or an error that prevents saving.

I submitted this S/O question a while ago about trying to get this behavior in a custom block, but as I say above I think it would be nice to have this a global option that could be enabled (at least for built-in components):
https://stackoverflow.com/questions/68956591/disable-wordpress-editor-update-button-if-color-selections-fail-color-contrast-c

**Comment by carolinan** [comment]
The theme can turn off custom colors and in worst case this would leave the user unable to save.
I think that this feature is better suited as a plugin,

**Comment by 00travelgirl00** [comment]
I agree with @carolinan as I experienced already a false warning from the color contrast checker. It would be a big probem, if the user would not be able to save this post. 

see also  https://github.com/WordPress/gutenberg/issues/20508
